src/modules/helpchan.ts

```diff
@@ -39,6 +41,19 @@ This channel will be dedicated to answering your question only. Others will try
 For more tips, check out StackOverflow's guide on **[asking good questions](https://stackoverflow.com/help/how-to-ask)**.
 `;
 
+const occupiedMessage = (asker: User) => `
+**This channel is claimed by ${asker.toString()}**
+
+This channel is dedicated to answering their question only (and any of their follow-up questions). Others will try to answer and help solve the issue.
+
+**To ${asker.toString()}, keep in mind:**
+• It's always ok to just ask your question. You don't need permission.
+• Explain what you expect to happen and what actually happens.
+• Include a code sample and error message, if you got any.
```

Comment:
I kept these the same as in the available embed, but maybe we should update them to be things we frequently tell people?

- `\``ts for code block syntax highlighting
- https://typescriptlang.org/play and https://tsplay.dev
- Don't expect an answer immediately, but you'll usually get an answer within a few hours; if you don't, your question is probably hard to answer — consider adding more details.

Reply:
Those sound good, but I think that would be better in a separate PR that updates both texts

Reply:
Sounds good

Reply:
I'm also wary of adding too many more strings with #91 seems close to merging